src/setup.py

```diff
@@ -5,8 +5,8 @@
 
 extensions = [
     Extension(
-        "pdist_agregation",
-        sources=["_pdist_agregation.pyx"],
+        "pdist_aggregation",
+        sources=["_pdist_aggregation.pyx"],
         include_dirs=[numpy.get_include()],
         extra_compile_args=['-fopenmp'],
         extra_link_args=['-fopenmp'],
```

Comment:
I got this error during the build: `clang: error: unsupported option '-fopenmp'` (on MacOS).

A hotfix for me was to add the `-Xpreprocessor` flag.

```
extensions = [
    Extension(
        "pdist_aggregation",
        sources=["_pdist_aggregation.pyx"],
        include_dirs=[numpy.get_include()],
        extra_compile_args=["-Xpreprocessor", "-fopenmp"],
        extra_link_args=["-Xpreprocessor", "-fopenmp"],
        define_macros=[("NPY_NO_DEPRECATED_API", "NPY_1_7_API_VERSION")],
    )
]
```

Reply:
Did OpenMP come when installing? You can check with:
```bash
conda list -n pdist_aggregation
```

Reply:
It seems that it did:

```
$ conda list -n pdist_aggregation | grep openmp
libopenblas               0.3.15          openmp_h5e1b9a4_0    conda-forge
llvm-openmp               11.1.0               hda6cdc1_1    conda-forge
```

Reply:
Which `clang` are you using?

See this comment regarding `llvm` and OpenMP on Mac: https://github.com/microsoft/LightGBM/issues/3#issuecomment-332025880

Reply:
I made it work using the [`compilers`](https://anaconda.org/conda-forge/compilers) package (see [64da803](https://github.com/scikit-learn-inria-fondation/pdist_aggregation/pull/6/commits/64da8033cf798148519aa671d51c386b7a9a41e1)).

Thanks @ogrisel!

Apparently the system `clang` in does not support `OpenMP` by default on MacOS.